### ISSUE TITLE
Add approval buttons to transducer tracking and planning modules

### ISSUE THREAD

**Issue by ebrahimebrahim** [OpenwaterHealth/SlicerOpenLIFU#113]
Besides virtual fit approval, described in #84, there are two additional types of approval.

## Transducer tracking approval

**How approval happens:** A button in the transducer tracking module can be clicked to approve transducer tracking. This writes to an in-memory `bool` attribute `transducer_tracking_approved` in the `openlifu.Session`.

**How approval is revoked:** It is revoked if the transducer is moved. The approve button should also become an "unapprove" button once the approval is there, so that theyuser can manually unapprove.

**What approval affects:** When there is an active session and transducer tracking is not yet approved, a red warning is displayed in the planning module to alert you not to use it. (In guided mode, you shouldn't even be able to get to the planning module without approving transducer tracking. And if you did somehow get there you should be blocked from planning.)

## Solution approval

**How approval happens:** After you run the planning and generate a solution, there is an approve button which becomes enabled to approve the solution. This writes to an `approved` attribute on the Solution object, which will ultimately be how a SlicerOpenLIFUSolution works once #96 is done.

**How approval is revoked:** If the planning module is used to generate a new plan, that will be a unique and new solution, so it will start in a default state of being not approved. It will replace the previous solution so there is no need to ever automatically unapprove a solution. The approve button can become an "unapprove" button once the approval is there, so that the user can manually unapprove.

**What approval affects:** In the sonication control module, one can only proceed if the solution was approved.

## Dependencies and notes

Depends on #96 
Depends on https://github.com/OpenwaterHealth/OpenLIFU-python/issues/95
